src/app/lib/analyticsUtils/chartbeat.js

```diff
@@ -0,0 +1,39 @@
+export const chartbeatUID = 50924;
+
+export const useCanonical = true;
+
+export const cknsSylphid = () => {};
+
+export const domain = service => {
+  const serviceLower = service.toLowerCase();
+
+  return serviceLower === 'news' ? 'bbc.co.uk' : `${service}.bbc.co.uk`;
+};
+
+const buildProducerArr = ({ service, producer }, type) => [
+  `${service} - ${producer}`,
+  `${service} - ${producer} - ${type}`,
+];
+
+const buildChapterArr = ({ service, chapter }, type) => [
+  `${service} - ${chapter}`,
+  `${service} - ${chapter} - ${type}`,
+];
+
+const buildServiceType = ({ service }, type) => [`${service} - ${type}`];
+
+export const sections = x => {
```

Comment:
I'd suggest passing in pageType, service, producer and chapter at the top level, rather than nesting them in an object x. 

```suggestion
export const sections = (pageType, service, chapter = '', producer = '', ) => {
```